docs/solutions/macos/scripts/generate-printer-install-scripts.sh

```
@@ -0,0 +1,52 @@
+divider="--------------------------------------------------------------------------------"
+yamltxt="software:
+  packages:"
+
+if [ -z "$csvpath" ] || [ ! -r "$csvpath" ]
+then
+    echo "Usage: $0 <printers.csv>. File missing or unreadable. Exiting..." >&2; exit 1
+fi
+
+# generate one install script per printer, and collect the packages.yml block
+{
+    read -r header
```

Shellcheck (0.11.0)

[warning] header appears unused. Verify use (or export if used externally).

(SC2034)